tmt/cli.py

```diff
@@ -772,6 +772,8 @@ def tests_import(
         tmt.convert.read_manual(plan, case, disabled, with_script)
         return
 
+    dry_run: bool = kwargs.get('dry', False)
```

Comment:
I'd recommend making it a function parameter. That would give it a proper type annotation. We already have 16 parameters, we can easily live with 17.